AIproject/kingandassassins.py

```diff
@@ -9,7 +9,7 @@
 from lib import game
 
 CARDS = {
-    # (PA King, PA Knight, Cuff, PA Population/Assassins
+    # (AP King, AP Knight, Fetter, AP Population/Assassins)
```

Comment:
I also changed these to match the terms used in the [English rule book](http://galakta.pl/download/K&A_rulebook_ENG_BGG.pdf).